bin/journey-test.sh

```diff
@@ -269,18 +269,24 @@ main() {
 
   clean "${build_dir}"
 
-  # Make a local version of trackler that includes the source from this repo.
-  git_clone "x-api" "${xapi_home}"
-  git_clone "trackler" "${trackler_home}"
-  assert_ruby_installed "${trackler_home}"
+  # Download everything we need in parallel
+  git_clone "x-api" "${xapi_home}" &
+  git_clone "trackler" "${trackler_home}" &
+  download_exercism_cli $(get_operating_system) $(get_cpu_architecture) "${exercism_home}" &
+  wait
+
+  # Check and install the ruby stuff we need in parallel
+  assert_ruby_installed "${trackler_home}" &
+  assert_ruby_installed "${xapi_home}" &
+  wait
```

Comment:
Not necessarily after `download_exercism_cli` but they need to happen after `git_clone "x-api" "${xapi_home}"` and `git_clone "trackler" "${trackler_home}"` so that they can verify the version of ruby that is required to build the tracker and x-api